src/System.Windows.Forms/src/System/Windows/Forms/TabControl.TabControlAccessibleObject.cs

```diff
@@ -0,0 +1,187 @@
+                // The "NativeMethods.CHILDID_SELF" constant returns to the id of the trackbar,
+                // which allows to use the native "get_accState" method to get the "State" property
+                => GetSystemIAccessibleInternal()?.get_accState(NativeMethods.CHILDID_SELF) is object accState
+                    ? (AccessibleStates)accState
+                    : AccessibleStates.None;
+
+            internal override IRawElementProviderFragmentRoot FragmentRoot => this;
+
+            internal override bool IsSelectionRequired => true;
+
+            internal override int[]? RuntimeId
+            {
+                get
+                {
+                    if (!_owningTabControl.IsHandleCreated)
+                    {
+                        return base.RuntimeId;
+                    }
+
+                    var runtimeId = new int[2];
+                    runtimeId[0] = RuntimeIDFirstItem;
+                    runtimeId[1] = PARAM.ToInt(_owningTabControl.InternalHandle);
+
+                    return runtimeId;
+                }
+            }
+
+            public override AccessibleObject? GetChild(int index)
+            {
+                if (!_owningTabControl.IsHandleCreated || _owningTabControl.TabPages.Count == 0
+                    || index < 0 || index > _owningTabControl.TabPages.Count)
+                {
+                    return null;
+                }
+
+                return index == 0
+                    ? _owningTabControl.SelectedTab.AccessibilityObject
+                    : _owningTabControl.TabPages[index - 1].TabAccessibilityObject;
+            }
+
+            public override int GetChildCount()
+            {
+                if (!_owningTabControl.IsHandleCreated)
+                {
+                    return -1;
+                }
+
+                if (_owningTabControl.TabPages.Count == 0)
+                {
+                    return 0;
+                }
+
+                return _owningTabControl.TabPages.Count + 1;
+            }
+
+            public override AccessibleObject? HitTest(int x, int y)
+            {
+                if (!_owningTabControl.IsHandleCreated)
+                {
+                    return null;
+                }
+
+                Point point = new(x, y);
+                if (_owningTabControl.SelectedTab.AccessibilityObject.Bounds.Contains(point))
+                {
+                    return _owningTabControl.SelectedTab.AccessibilityObject;
+                }
+
+                foreach (TabPage tabPage in _owningTabControl.TabPages)
+                {
+                    if (tabPage.TabAccessibilityObject.Bounds.Contains(point))
+                    {
+                        return tabPage.TabAccessibilityObject;
+                    }
+                }
+
+                return null;
+            }
+
+            internal override UiaCore.IRawElementProviderFragment? ElementProviderFromPoint(double x, double y)
```

Comment:
Remove extra difinitions here and in the rest places of your fix:
```suggestion
            internal override IRawElementProviderFragment? ElementProviderFromPoint(double x, double y)
```